ocs_ci/utility/powernodes.py

```diff
@@ -56,6 +57,37 @@ def verify_machine_is_down(self, node):
         else:
             return True
 
+    def stop_powernodes_machines_powervs(
+        self, powernode_machines, timeout=900, wait=True, force=True
+    ):
+        """
+        Stop PowerNode Machines - PowerVS
+
+        Args:
+            powernode_machines (list): PowerNodes objects
+            timeout (int): time in seconds to wait for node to reach 'not ready' state
+            wait (bool): True if need to wait till the restarted node reaches timeout
+                         - for future use
+            force (bool): True for PowerNode ungraceful power off, False for
+                graceful PowerNode shutdown - for future use
+
+        Raises:
+            UnexpectedBehaviour: If PowerNode machine is still up
+        """
+        for node in powernode_machines:
+            cmd = f"ssh core@{node.name} sudo systemctl stop kubelet.service"
+            result = exec_cmd(cmd)
```

Comment:
So basically this will be moved out of here to service.py module.  And I would then also check the real status of the service there if it's really stopped.